proposals/notifications.md

```diff
@@ -0,0 +1,88 @@
+# Solid Notifications
+
+Solid notifications are intended to announce social interactions and data. A Solid notification is an [LDP Resource](https://www.w3.org/TR/ldp/#ldpr) which is submitted to and resides in an [LDP Container](https://www.w3.org/TR/ldp/#ldpc), i.e., a [Solid inbox](inbox.md).
+
+The contents of a notification is application specific. We derive two core principles from this:
+
+1. a notification can contain arbitrary data
+2. a notification is vocabulary agnostic
+
+Applications can create notifications to use any vocabulary to describe its contents in any shape and form. Multiple notifications can be contained inside a single resource or span across different resources. A single resource with multiple notifications may be suitable for `HTTP PATCH`, whereas multiple resources containing a single notification may be suitable for `HTTP POST` or `PUT`.
+
+
+## Discovery
+As notifications are contained in inboxes, a Solid inbox container has `ldp:contains` relations to each notification resource. If a [Solid outbox](outbox.md) is employed, it may also contain references to the notification.
```

Comment:
Yes, future PR. We could leave the link out. Most importantly, it is an example in which something (like an outbox, which is a reasonable candidate) has a trace of - a link to - the notification.